src/test/java/org/jabref/logic/net/MimeTypeDetectorTest.java

```diff
@@ -31,35 +32,29 @@ public void handlePermanentRedirections() {
                 )
         );
 
-        assertTrue(MimeTypeDetector.isPdfContentType(redirectedUrl));
+        assertTrue(new URLDownload(redirectedUrl).isMimeType("application/pdf"));
```

Comment:
Introduce the shortcut method `isPdf` or create an enum for the mime types? I don't like the magic strings...